intera_interface/src/intera_interface/limb.py

```diff
@@ -197,6 +229,9 @@ def _on_endpoint_states(self, msg):
             ),
         }
 
+    def _on_tip_states(self, msg):
+        self._tip_states = deepcopy(msg)
```

Comment:
Not completely sure how interrupts happens in python, I wanted to make sure that the message was not modified while being used by something else. 

It seems like the only other thing using the _tip_states variable for now is the tip_state() function which does a deepcopy of for a single EndpointState from the list.